tests/ccgbugs/xarray9578.nim

```diff
@@ -1,7 +1,8 @@
-import t9578
+import tcodegen_generic_openarray_import
 
 proc testArray*(x: var array[3,mytype]) =
   f(x[0].addr)
 
 proc testArray2*(x: var ptr array[3,mytype]) =
   f(x[0].addr)
+
```

Comment:
Some files end with a new-line and some don't. I think this should be done consistently, but I'm not sure which style is preferred.